velox/common/memory/MappedMemory.h

```diff
@@ -26,12 +26,88 @@
 #include <gflags/gflags.h>
 #include "velox/common/base/Exceptions.h"
 #include "velox/common/memory/MemoryUsageTracker.h"
+#include "velox/common/time/Timer.h"
 
 DECLARE_bool(velox_use_malloc);
 DECLARE_int32(velox_memory_pool_mb);
 
 namespace facebook::velox::memory {
 
+struct SizeStats {
```

Comment:
Would you add comments to explain each of these stats? In particular, it is not clear what are xxxClocks stats.